### ISSUE TITLE
callIfFunction, applyIfFunction

### ISSUE THREAD

**Issue by tommmyy** [tommmyy/ramda-extension#125]
```js
const ref = { current: null };
const getRef = always(ref);
const createRef = (current) => ({ current });

applyIfFunction(ref, []); // { current: null }
applyIfFunction(getRef, []); // { current: null }
applyIfFunction(createRef, [document.body]); // {"current": [object HTMLBodyElement]}

callIfFunction(ref) // { current: null }
callIfFunction(getRef) // { current: null }
callIfFunction(createRef, document.body) // {"current": [object HTMLBodyElement]}
```

I ofter use the function in React components. 
Sometimes a property is either function (then you need to call it) or just static value.

**Comment by wafflepie** [comment]
Love the idea!

**Comment by tommmyy** [comment]
closed by #126
